pom.xml

```diff
@@ -69,6 +69,7 @@
     <jacoco-plugin.version>0.8.6</jacoco-plugin.version>
     <project-info-reports-plugin.version>3.1.1</project-info-reports-plugin.version>
     <maven-deploy-plugin.version>3.0.0-M1</maven-deploy-plugin.version>
+    <dockerfile-maven-plugin.version>1.4.13</dockerfile-maven-plugin.version>
```

Comment:
Since `dockerfile-maven-plugin` / [`dockerfile-maven`](https://github.com/spotify/dockerfile-maven) is inactive / not actively developed anymore I would prefer to use some other tools for docker image generation, e.g. [Jib](https://github.com/GoogleContainerTools/jib)

Reply:
I was thinking about this as well, but didn't found a proper solution for copying specific files to the build context. I checked again and decided to get rid of the files (e.g. `docker-entrypoint.sh`) and updated to the jib plugin.